docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -450,6 +450,8 @@ To set up Generic OAuth authentication with Descope, follow these steps:
 
 ### Set up OAuth2 with Auth0
 
+> **Note:** Support for the Auth0 "audience" feature is not currently available in Grafana. For roles and permissions, the available options are described [here]({{< relref "../../../../administration/roles-and-permissions/" >}}).
```

Comment:
afaik we use the following format for notes, @irenerl24 could you please confirm? 
```suggestion
{{% admonition type="note" %}}
Support for the Auth0 "audience" feature is not currently available in Grafana. For roles and permissions, the available options are described [here]({{< relref "../../../../administration/roles-and-permissions/" >}}).
{{% /admonition %}}
```

Reply:
Yes, we are using this format for notes :)